src/components/Calender/DateDisplay.tsx

```diff
@@ -67,7 +67,7 @@ export const DateDisplay = (props: DateDisplayProps) => {
                     isToday={isToday}
                     isCurrentMonth={isCurrentMonth}
                     isSelected={isSelected}
-                    onPressHandler={onDateTilePressed}
+                    onPressHandler={props.onDateTilePress}
```

Comment:
lool with the name onPressHandler, could be handlePress, instead of a "press handler" what even is "press handler"???